packages/app/src/game/module/board/overlays/readyOverlay.tsx

```diff
@@ -0,0 +1,62 @@
+import { getUserId } from "packages/app/src/auth";
+import { PlayerListPlayer, StreakType } from "packages/models/lib";
+import * as React from "react";
+import { useSelector } from "react-redux";
+import { AppState } from "../../../../store";
+import { BoardOverlay } from "./boardOverlay";
+
+const getStreakType = (player: PlayerListPlayer): string => {
+
+    const streakType = player?.streakAmount === 0 ? ""
```

Comment:
Please use an if statement here for readability. Ternaries are great for a single statement but beyond that I find them hard to read (even though you've done a nice job with the line breaks)

```ts
if (!player || player.streakAmount === 0) {
    return "";
}

if (player.streakType === StreakType.WIN) {
    return player.streakAmount === 1 ? "Win" : "Wins";
}

return player.streakAmount === 1 ? "Loss" : "Losses"
```

Reply:
*Changed to an if statement for getStreakType